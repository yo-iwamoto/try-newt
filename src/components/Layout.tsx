import type { ReactNode, VFC } from 'react';

type Props = {
  children: ReactNode;
};

export const Layout: VFC<Props> = ({ children }) => <div>{children}</div>;
