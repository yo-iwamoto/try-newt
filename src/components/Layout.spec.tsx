import { Layout } from './Layout';
import { render } from '@testing-library/react';

describe('[component] Layout', () => {
  it('renders Layout unchanged', () => {
    const { container } = render(
      <Layout>
        <h1>Hello World!</h1>
      </Layout>
    );

    expect(container).toMatchSnapshot();
  });
});
