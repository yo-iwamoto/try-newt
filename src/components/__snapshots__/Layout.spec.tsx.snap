// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[component] Layout renders Layout unchanged 1`] = `
<div>
  <div>
    <h1>
      Hello World!
    </h1>
  </div>
</div>
`;
